package.json

```diff
@@ -30,10 +30,13 @@
     "babel-core": "^6.24.1",
     "babel-preset-es2015": "^6.24.1",
     "babel-preset-react": "^6.24.1",
+    "babel-preset-stage-2": "^6.24.1",
     "babel-require": "^1.0.1",
     "deepmerge": "^1.3.2",
     "enzyme": "^2.8.2",
     "expect": "^1.20.2",
+    "jsdom": "^10.1.0",
+    "jsdom-global": "^3.0.2",
```

Comment:
jsdom is required to use `mount()` afaik

(maybe not required, but easiest solution anyways)